portfolio/src/main/webapp/game.js

```diff
@@ -46,9 +51,75 @@ class Board {
   updateFrame(xPos, yPos) {
     // clear old frame
     ctx.clearRect(0, 0, canvas.width, canvas.height);
-    this.drawBall(xPos, yPos);
+    this.determineSquare(xPos, yPos);
+    this.drawBoard();
     this.drawGrid();
   }
+
+  determineSquare(xPos, yPos) {
+    var col = 0;
```

Comment:
Nit:
IMO a better name is something along the lines of markSquare or UpdateState since this function determines the square and also updates the state/model which you then use in drawBoard.